Webpack errors should fail the export command

### What's wrong?

When running the regular dev command you will see Webpack errors like `Module not found: Error: Can't resolve......`, but the export command completely hides that and from the console output it looks like a successful build.

### Steps to reproduce

_(I'm running this on latest version, 4.6.4)_

1. Introduce a bug where a file is importing another file that doesn't exist.
2. `cosmos-export`
3. The build should now fail and display the errors, but instead the output looks the same as a successful build.

### Additional context

I looked into it a bit and it looks like we might want to check the `stats` object here: https://github.com/react-cosmos/react-cosmos/blob/master/packages/react-cosmos/src/server/web/export.js#L31 (webpack docs: https://webpack.js.org/api/node/#stats-haserrors-). Unfortunately I ran into build issues with Babel when trying to get the `react-cosmos` repo running locally, and I don't have time currently to dig deeper to fix it. But maybe someone else can pick up from here :)

--- a/.codecov.yml
+++ b/.codecov.yml
@@ -6,7 +6,3 @@ coverage:
       default:
         target: 90%
         threshold: 5%
-    patch:
-      default:
-        target: 80%
-        threshold: 25%

--- a/packages/react-cosmos/src/server/web/export.js
+++ b/packages/react-cosmos/src/server/web/export.js
@@ -31,6 +31,11 @@ const runWebpackCompiler = (webpack, config) =>
     compiler.run((err, stats) => {
       if (err) {
         reject(err);
+      } else if (stats.hasErrors()) {
+        const error = new Error('Errors occurred');
+        //$FlowFixMe
+        error.webpackErrors = stats.toJson().errors;
+        reject(error);
       } else {
         resolve(stats);
       }
@@ -87,8 +92,15 @@ export async function generateExport() {
         console.log(outputPath);
       },
       err => {
-        console.error('[Cosmos] Export Failed! See error below:');
-        console.error(err);
+        console.error('[Cosmos] Export Failed! See errors below:\n');
+        if (err.webpackErrors) {
+          err.webpackErrors.forEach(error => {
+            console.error(`${error}\n`);
+          });
+        } else {
+          console.error(err);
+        }
+        process.exit(1);
       }
     );
 }

